docs/diagrams/AddMcqSequenceDiagram.puml

```diff
@@ -0,0 +1,80 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":QuickCacheParser" as QuickCacheParser LOGIC_COLOR
+participant ":AddMultipleChoiceQuestionCommandParser" as AddMultipleChoiceQuestionCommandParser LOGIC_COLOR
+participant "t:AddMultipleChoiceQuestionCommand" as AddMultipleChoiceQuestionCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "QuickCache" as QuickCache MODEL_COLOR
+participant "flashcards:UniqueFlashcardList" as Flashcards MODEL_COLOR
+participant "internalList:ObservableList<Flashcard>" as ObservableList MODEL_COLOR
+end box
+[-> LogicManager : execute(add q/question ans/1 c/first c/second)
```

Comment:
Lets standardize to `"add q/question ans1 c/first c/second"` (with the inverted commas)

Reply:
Yup, changed alr!